client/static/js/main.js

```diff
@@ -251,9 +252,8 @@ function renderGraph(pk){
                     $('#qnet-graph').bind('plotselected', (event, ranges) => zoom(ranges, qnetOptions, (opts) => plotQnet('#qnet-graph', 'qnet', qnetOptions)));
                     $('#qnet-graph').bind('plothover', (event, pos, item) => hover(event, pos, item, 'Conc.: ', ' µM', 'qNet: ', ' C/F', '#hoverdata'));
                     $('#qnet-graph').mouseout((event)=>hoverOut('Conc.: ', ' µM', 'qNet: ', ' C/F', '#hoverdata'));
-                }else{
-                    // hide qnet button
-                    $('#qnet').hide();
+					
+					$('#qnet').show();
```

Comment:
Logically this doesn't make sense. It looks like if there's no qNet data you want to show the qNet button!

Reply:
It used to be as follows
```
                if(graphData['qnet'].length > 0){
                   ....
                }else{
                    // hide qnet button
                    $('#qnet').hide();
                }
```
but since I change the default to be hidden it needs the show inside the if clause and the else calsue is redundant. If you expand you'll see that that show is inside the if clause checking that there is in fact qnet data and that teh else clause is gone alltogether.